### PR TITLE
Add clipping polygon support

### PR DESCRIPTION
**Pull request by wazolab**

### CODE REVIEW

File: components/Home/Embedded.vue [hunk 1]
```diff
@@ -130,6 +130,7 @@ watch(selectedCategoryIds, (a, b) => {
       menuStore.fetchFeatures({
         vidoConfig: config!,
         categoryIds: selectedCategoryIds.value,
+        clipingPolygonSlug: route.query.clipingPolygonSlug?.toString() || undefined,
```
Comment (frodrigo) [inline components/Home/Embedded.vue:133]: undefined is not already the default ?

Comment (wazolab) [reply components/Home/Embedded.vue:133]: right :)

File: components/Home/Home.vue [hunk 2]
```diff
@@ -308,6 +309,7 @@ async function fetchFavorites() {
 
   return await getPois(config!, favoritesIds.value, {
     geometry_as: 'point',
+    cliping_polygon_slug: route.query.clipingPolygonSlug?.toString() || undefined,
```
Comment (frodrigo) [inline components/Home/Home.vue:312]: Same here

File: components/PoisList/Actions.vue [hunk 3]
```diff
@@ -48,6 +48,7 @@ export default defineNuxtComponent({
           geometry_as: 'point',
           short_description: false,
           format,
+          cliping_polygon_slug: this.$route.query.clipingPolygonSlug?.toString() || undefined,
```
Comment (frodrigo) [inline components/PoisList/Actions.vue:51]: here

File: pages/pois/[ids]/map.vue [hunk 4]
```diff
@@ -35,6 +36,7 @@ if (params.ids) {
   const getPoiPromise = getAsyncDataOrThrows('getPoiPromise', async () =>
     await getPois(config.value!, ids, {
       geometry_as: undefined,
+      cliping_polygon_slug: route.query.clipingPolygonSlug?.toString() || undefined,
```
Comment (frodrigo) [inline pages/pois/[ids]/map.vue:39]: again

**Pull request merged**